src/sections/NavBar/NavBar.scss

```diff
@@ -12,23 +12,23 @@
   }
 
   &-content {
-    margin-left: 100px;
     height: 100%;
     width: $width-lg-max;
     display: flex;
     justify-content: space-between;
     align-items: center;
     list-style: none;
-    @include tablet {
-      margin: 0 30px;
-    }
     @include phone {
       justify-content: space-between;
     }
   }
-  
+
   &-links-container {
     display: flex;
+    margin: 0 100px;
+    @include tablet {
+      margin: 0 30px;
+    }
   }
```

Comment:
this is what I added. sorry my auto formatter makes it look like i changed a lot of code.